### PR TITLE
HBW-181 Integrate new claiming components

### PR DESCRIPTION
**Pull request by vladdoncheg**

### CODE REVIEW

**Comment by vladdoncheg** [comment]
Скриншот 2019-10-16 10 50 17

File: app/views/layouts/_navigation.html.haml [hunk 1]
```diff
@@ -26,6 +26,10 @@
               = link_to users_path do
                 %span.fas.fa-users
                 = t('.users')
+            %li
+              = link_to tasks_path do
+                %span.fas.fa-check-double
```
Comment (TimAle) [inline app/views/layouts/_navigation.html.haml]: https://fontawesome.com/icons/tasks?style=solid

**Comment by vladdoncheg** [comment]
- Removed rendering old button and list;
- Integrated new button and list;
- Fixed 'View all' path;
- Remade icon. Now it's a component, has right color and available to hover.


Скриншот 2019-10-17 12 48 01

Hovering:
Скриншот 2019-10-17 12 52 43

Opened pop_up:
Скриншот 2019-10-17 12 47 32

**Comment by vladdoncheg** [comment]
Also:
- Simplyfied integrating button;
- Removed blinking 'No tasks' while fetching.

**Pull request merged**